packages/terra-consumer-nav/src/components/modal/Modal.scss

```diff
@@ -5,13 +5,15 @@
     background: var(--terra-consumer--modal-background, #fff);
     border-radius: 5px;
     left: 20px;
+    max-height: calc(100% - 80px);
```

Comment:
where does 80px come from?

Reply:
Modal header height is 60px and fullscreen Modal has 10px/20px below the content for mobile/table.